### DISCUSSION TITLE
ChampionshipPrediction endpoint

### DISCUSSION THREAD

**Discussion by simone142** [br-g/openf1#3] [Ideas]
Add the `ChampionshipPrediction` endpoint, already provided on `livetiming.formula1.com`

**Comment by br-g** [comment]
Hello, thanks for your suggestion.
I will have a look at this endpoint!

**Comment by dauble** [comment]
Would also love to see this. An example can be found here:
https://livetiming.formula1.com/static/2024/2024-04-07_Japanese_Grand_Prix/2024-04-07_Race/ChampionshipPrediction.json
